CIP-0061/README.md

```diff
@@ -0,0 +1,77 @@
+---
+CIP: 61
+Title: Stake-based Protocol Governance Restrictions
+Authors: Matthias Benkort <matthias.benkort@cardanofoundation.org> 
+Discussions-To: 
+Comments-Summary: A simple, transitional, veto power for ada holders until decentralized governance.  
+Comments-URI:  
+Status: Proposed 
+Type: Standards Track 
+Created: 2022-06-25
+License: CC-BY-4.0
+---
+
+## Abstract
+
+The Cardano journey was divided into several phases, the last of which relates to the decentralization of the network governance. We call this last phase: 'Voltaire'. While Voltaire is in the making, some aspects of the protocol remain relatively centralized and under the control of a few entities. This proposal improves on the current status quo by adding a layer of control to the Stake Pool Operators (a.k.a. SPOs) and by extension, to delegators. This proposal is fairly lightweight and can be implemented without a hard fork. 
+
+## Motivation
+
+Decentralized governance is a hard topic and it is likely to take some time to get to a point where Cardano has a fully working decentralized governance. In the meantime, while tolerated until now, the current status quo is judged unacceptable by many. As of today, the number of entities that can control the current set of governance operations is limited. Those operations include:
+
+- Protocol parameters updates
+- Protocol versions updates
+- Transfers to and from the reserve (a.k.a MIR transfers)
+- Transfers to and from the treasury (a.k.a MIR transfers)
+
+At this stage, any such operation can be performed if and only if a [quorum of _5 out of 7_](https://github.com/input-output-hk/cardano-configurations/blob/master/network/mainnet/genesis/shelley.json#L57)<sup>\*</sup> genesis members is reached. Genesis members represent here keys that were assigned to the three founding entities: Input Output, Emurgo and the Cardano Foundation. Because the current quorum limit is 5 and because of the current distribution of the genesis key (3 - 2 - 2), it suffices for only 2 out of the 3 funding entities to perform such an operation. 
+
+Over the past months and years, this topic has been at the centre of many vigorous debates. We thus think that the current status quo has to change.
+
+> \* This ratio is currently configurable via a protocol parameter.
```

Comment:
the quorum number is not a protocol parameter (at least in the sense that we use that term in the ledger). it cannot be updated by the protocol parameter update system. it's set once in the genesis config.

Reply:
indeed, it is a global variable (and hence requires a hardfork to change).

Reply:
https://github.com/input-output-hk/cardano-ledger/blob/0913292b13963ae4b60136eddb8d18b137f96a21/libs/cardano-ledger-core/src/Cardano/Ledger/BaseTypes.hs#L601-L602